python-malformed-yaml/github_service.py

```diff
@@ -0,0 +1,20 @@
+from github import Github
+
+
+class GitHubService():
```

Comment:
This doesn't quite sound right. When we create a GitHubService we are passing the repository name and pr number. I'm wondering if we need to call this something like `GitHubPullRequest`  instead, what do you think?

Reply:
If I understand rightly you mean it's not a true GitHubService as it's just reading some PR info and writing a PR comment (`get_changed_files_from_pr` and `fail_pr`)

Reply:
renamed GitHubPullRequest and corresponding renamings